src/main/java/Main.java

```diff
@@ -1,6 +1,55 @@
+import java.util.ArrayList;
+import java.util.Scanner;
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+        Scanner scanner = new Scanner(System.in);
+        int amountOfGuests = -1;
```

Comment:
```suggestion
        int numberOfGuests = -1;
```
NIT: amount употребляется с неисчисляемыми